parameter-sets/connection-info-v2/parameter-set.json

```diff
@@ -0,0 +1,75 @@
+{
+    "meta" : {
+        "label" : "AKS connections",
+        "pluralLabel": "AKS connections",
+        "description" : "AKS connection settings",
+        "icon" : "icon-link"
+    },
+    
+    "defaultDefinableAtProjectLevel" : true,
+    "defaultDefinableInline" : true,
+
+    "pluginParams": [
+    ],
+
+    "params": [
+        {
+            "name": "identityType",
+            "label": "Identity type",
+            "description": "Select the type of identity",
+            "type": "SELECT",
+            "selectChoices": [
+                {"value": "default", "label": "Default credentials, from environment or System Assigned Managed Identity"},
+                {"value": "user-assigned-client-id", "label": "User Assigned Managed Identity - Client Id"},
```

Comment:
gievn that clientId and resourceId can easily be told apart, we shouldn't have both "user-assigned-client-id" and "user-assigned-resource-id", but a single "user-assigned-id" and detect which type it is in python (clientId is a UUID, easy to check with a regex)

Reply:
Good idea